data/human/republic dump intro.txt

```diff
@@ -0,0 +1,1034 @@
+
+mission "Joining the Navy"
+	autosave
+	name "Navy Candidate"
+	description "Destroy the pirate vessel named <npc> near the <system> system and return to <planet> for an opportunity to join the Republic Navy."
+	source
+		government "Republic"
+	destination "Geminus"
+	on fail
+		dialog `You have failed an essential Navy mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	to offer
+		has "event: war begins"
+		has "event: enlistment begins"
+#		not "event: naval employment earned"
+		has "enlistment announced: offered"
+		not "chosen sides"
+	on offer
+		conversation
+			`Your ship's communicator lights with an official announcement from the Republic:`
+			``
+			`	"Hello, Captain <last>," the message begins, "as of now, the Republic Navy is seeking captains skilled in combat to serve in the Navy. However, to discourage people hoping for handouts, applicants need to prove that they want to serve in the Navy. Recently, we've been having issues with a pirate vessel called the <npc> - our intel reports that it was spotted skulking around the <system> system. Destroy the <npc>, and return to <destination> to be formally accepted into the Navy."`
+			choice
+				`	I'll take on that ship.`
+					goto jobAccept
+				`	Sorry, but I have no desire to join the Navy at this time.`
+					defer
+			label jobAccept
+			`	"We look forward to seeing you on <destination> once the mission is over," the message reads. "Good luck!"`
```

Comment:
You may want to consider the usage of "Good Hunting." It has some significance in the US military and would likely fit this much better.

Reply:
Hmm, but would a galactic republic remember or care about the military traditions of a 1000 year old country?

Reply:
This review comment is on the dump file, i.e. the file where everything in it is being reworked and replaced...